python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -530,25 +529,39 @@ def is_generic_test_on_attached_node_from_dbt_resource_props(
 def default_asset_check_fn(
     asset_key: AssetKey,
     unique_id: str,
-    dagster_dbt_translator_settings: "DagsterDbtTranslatorSettings",
+    dagster_dbt_translator: "DagsterDbtTranslator",
     dbt_resource_props: Mapping[str, Any],
+    dbt_nodes: Mapping[str, Any],
 ) -> Optional[AssetCheckSpec]:
     is_generic_test_on_attached_node = is_generic_test_on_attached_node_from_dbt_resource_props(
         unique_id, dbt_resource_props
     )
 
     if not all(
         [
-            dagster_dbt_translator_settings.enable_asset_checks,
+            dagster_dbt_translator.settings.enable_asset_checks,
             is_generic_test_on_attached_node,
         ]
     ):
         return None
 
+    test_unique_id = dbt_resource_props["unique_id"]
+    # tests can't depend on tests, but include "test" so that we collect deps for test_unique_id
+    check_deps = get_deps(
```

Comment:
@rexledesma is this a reasonable use of get_deps? It's possible we don't need the ephemeral -> actual model resolution here

Reply:
Can we just figure this out using the `manifest` at [1]? This is essentially just all the extra models in `manifest["parent_map"][test_unique_id]`.